src/pages/Register/Register.jsx

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+// Hooks
+import { useFormik } from "formik";
+// Styles
+import "./Register.scss";
+// Components
+import { Typography, Button, Grid, TextField, Container, FormHelperText } from "@material-ui/core";
+
+import RegisterSchema from "./RegisterSchema";
+
+const Register = () => {
+  const formik = useFormik({
+    initialValues: {
+      name: "",
+      email: "",
+      password: "",
+    },
+    initialErrors: {
+      name: false,
+      email: false,
+      password: false,
+    },
+    initialTouched: {
+      name: false,
+      email: false,
+      password: false,
+    },
+    validationSchema: RegisterSchema,
+    handleSubmit(values) {
+      console.log(values);
+    },
+  });
+  return (
+    <Container className="outer-container">
+      <form className="inner-container" onSubmit={formik.handleSubmit}>
+        <Grid container alignItems="center" direction="row" justify="center">
+          <Grid item className="title-item">
+            <Typography variant="h1">Bienvenide a Healthy Dev</Typography>
+          </Grid>
+          <Grid item className="name-item">
+            <TextField
+              fullWidth
+              required
+              aria-describedby="Nombre"
+              error={formik.touched.name && formik.errors.name}
```

Comment:
Intercambiar valores para evitar error de tipos

```suggestion
error={formik.errors.name && formik.touched.name}
```